### DISCUSSION TITLE
Enable unused "/raid" command.

### DISCUSSION THREAD

**Discussion by Mickey42302** [PurpurMC/Purpur#1488] [__Completed__] [answered]
I have a question/suggestion. Recently, I heard about an unused "/raid" command in Vanilla which serves as a tool for managing raids. I decided to decompile the JAR so I could get a look at the code for this command. Sure enough, I found the Java class for it, but the command is inaccessible by default since it's not registered in the command list (as stated on TCRF). I would like to ask if it would be possible for Purpur to make the "/raid" command accessible? A command which allows for the management of raids would be very useful.

More information about the "/raid" command: https://tcrf.net/Minecraft:_Java_Edition#Unused_Commands

**Comment by RhythmicSys** [comment]
Would this be different from `/effect give bad_omen` ?  (Genuine question, wondering what the application would be)

**Reply by granny** [reply]
Giving yourself the `bad_omen` effect would be the same as running `/raid start`. I'm honestly surprised that it's disabled by default.

Documentation of the command: https://minecraft.wiki/w/Java_Edition_unused_features#/raid (OP lists a different wiki & i'm not sure how up-to-date it is compared to the "unofficially official" minecraft wiki)

**Comment by granny** [comment, marked as answer]
Implemented as a byproduct of build #2147 https://github.com/PurpurMC/Purpur/commit/8f371ac6c4f399d98d6dab29b755033753ef6314

> Set to true to register unused/hidden minecraft commands. The following commands become available (along with their `minecraft.command.<command_name>` permission counterpart): `debugconfig`, `serverpack`, `spawn_armor_trims`, `warden_spawn_tracker`, `debugmobspawning`, `debugpath`, `raid`, `resetchunks`, `test`. This list may or may not be incomplete depending on if they get removed or new ones get added in newer minecraft versions.
